### ISSUE TITLE
Adopt use of the Dataverse parent pom when available

### ISSUE THREAD

**Issue by qqmyers** [gdcc/dataverse-exporters#12]
Per https://github.com/IQSS/dataverse/issues/10482, we expect to make the Dataverse parent pom available for use in exporters. This issue is to update the example and current exporters to use it (which would keep them in sync with the versions of libraries used in Dataverse) once that is done, ~Jun '24. At that point, we may also look into publishing exporter jars to maven as well as a way to make them available for download and/or use in other projects.

**Comment by poikilotherm** [comment]
New plan coordinated with input by @qqmyers and @pdurbin:

- We create an exporters parent in this repo and publish it to Maven Central
- Exporters live in their own repos (e.g. gdcc/exporter-ddipdf, gdcc/exporter-crossaint, etc) and get released on their own
- Exporters reuse the released parent to align common dependencies and configuration
- This repo will contain a list of exporters and maybe have some docs how to create another exporter.
